### ISSUE TITLE
Version 1.3.4 has shipped without a /dist directory

### ISSUE THREAD

**Issue by jaredLunde** [GoogleChromeLabs/critters#48]
I've confirmed this on multiple devices and with both NPM + Yarn so I don't think this is just some weird failure of caching.

### ls -l node_modules/critters-webpack-plugin/  
```  
total 56
-rw-r--r--  1 jaredlunde  staff   1101 Jan  3 16:17 CONTRIBUTING.md
-rw-r--r--  1 jaredlunde  staff  11341 Jan  3 16:17 LICENSE
-rw-r--r--  1 jaredlunde  staff   5981 Jan  3 16:17 README.md
-rw-r--r--  1 jaredlunde  staff   2347 Jan  3 16:17 package.json
drwxr-xr-x  6 jaredlunde  staff    192 Jan  3 16:17 src
drwxr-xr-x  7 jaredlunde  staff    224 Jan  3 16:17 test
```

### cat node_modules/critters-webpack-plugin/package.json
```
{
  "name": "critters-webpack-plugin",
  "version": "1.3.4",
  "description": "Webpack plugin to inline critical CSS and lazy-load the rest.",
  "main": "dist/critters.js",
  "source": "src/index.js",
  ...
}
```

Further evidence seen here
- *1.3.4* https://unpkg.com/critters-webpack-plugin@1.3.4/dist/critters.js
- *2.5.0* https://unpkg.com/critters-webpack-plugin@2.5.0/dist/critters.js

**Comment by developit** [comment]
Yikes! I'll fix this today.

**Comment by developit** [comment]
Hi @jaredLunde - I really appreciate you letting me know! I've just published `critters-webpack-plugin@1.3.5` that fixes the issue ([notes](https://github.com/GoogleChromeLabs/critters/releases/tag/1.3.5)). Cheers!
